README.md

```diff
@@ -15,6 +15,40 @@ add the badge to your github project with the following snippet:
 [![Unitary Fund](https://img.shields.io/badge/Supported%20By-UNITARY%20FUND-brightgreen.svg?style=for-the-badge)](https://unitary.fund)
 ```
 
-## Writing a blog post
+## Writing a blog post 
+To add a blog post, you just need to add a markdown file (`YYYY_your_title.md`) under the `src/content/blog/` folder. 
+An example file is this [one on 2023 UF members](https://raw.githubusercontent.com/unitaryfund/unitary.fund/main/src/content/blog/2023_members.md), which renders [here](https://unitary.fund/posts/2023_members/). Make sure to edit the keys correctly, e.g., below
+```
+---
+title: Announcing the 2023 Unitary Fund Members
+author: Unitary Fund Team
+day: 7
+month: 2
+year: 2023
+---
+
+Since its launch last year...
+```
```

Comment:
Two things:

1. Do we need to use the full GitHub link? I think a local reference might work (and is more convenient for local IDE development).
2. What do you think about making this a generic template?

````suggestion
An example file is this [one on 2023 UF members](./src/content/blog/2023_members.md), which renders [here](https://unitary.fund/posts/2023_members/).
The following template post can be used to start from scratch:
```
---
title: post title
author: author one and author two
day: 7
month: 2
year: 2023
---

your markdown content
```
````